doc/HOWTO_oracle.md

```diff
@@ -57,92 +76,79 @@ sudo ln -s libocci.dylib.12.1   libocci.dylib
 sudo ln -s /opt/oracle/instantclient/sqlplus /usr/local/bin/sqlplus
 ```
 
-##### Add Environment Variables to profile
+### Set Up Environment Variables
 
-* Add to `~/.profile` (bash) or `~/.zprofile` (zsh)
+1. Add to `~/.profile` (bash) or `~/.zprofile` (zsh)
 
 ```
-export NLS_LANG="AMERICAN_AMERICA.AL32UTF8"
-export ORACLE_HOME=/opt/oracle/instantclient
```

Comment:
Are you sure we don't need `ORACLE_HOME` anymore? I think we need that in order to install the OCI gem.

Reply:
I think I forgot to update the instructions for installing instant client via `brew`—when I did that, everything gets put into `/usr/local`, like other brew-managed packages, and there is no need to specify these environment variables.

I’ll revisit the installation instructions for instant client.